src/server/mailgun.ts

```diff
@@ -43,10 +58,201 @@ export const sendWelcomeEmail = async (emailTo: string, fullName: string) => {
             template: 'welcome email',
             'h:X-Mailgun-Variables': JSON.stringify({
                 fullName,
-                resetPasswordLink: `https://${process.env.DOMAIN_NAME}/account/reset-password`,
+                resetPasswordLink: `${BASE_URL}/account/reset-password`,
             }),
         })
     } catch (error) {
-        logger.error.log('Welcome email error: ', error)
+        logger.error.log('sendWelcomeEmail error: ', error)
+    }
+}
+
+export const sendStudyProposalEmails = async (studyId: string) => {
+    const [mg, domain] = await mailGunConfig()
+    if (!mg) return
+
+    const { study, org } = await getStudyAndOrg(studyId)
+    const reviewersToNotify = await getUsersByRoleAndOrgId('reviewer', org.id)
+
+    for (const reviewer of reviewersToNotify) {
+        const email = reviewer.email
+        if (!email) continue
+
+        try {
+            await mg.messages.create(domain, {
```

Comment:
instead of creating a new message for each reviewer, can we make a single message and BCC each of them?  Or maybe we should use batch sending? https://help.mailgun.com/hc/en-us/articles/203068874-How-can-I-send-the-same-email-to-multiple-recipients

Reply:
BCC won't work in it's current state as each email has a unique "Hello <your_name>" in the email body... ideally we could remove that and just BCC and our lives would be a lot easier. Batch sending seems to have some extra work in the templates around [recipient variables](https://documentation.mailgun.com/docs/mailgun/user-manual/sending-messages/#recipient-variables) and may be even more work to maintain on that end

Reply:
let me check with iris on this to see if we can get away with BCC and no personalized text in the body

Reply:
ah, I'd missed that each email was personalized.  I think this is fine, we probably won't be sending even 100s at a time, but if the count does get larger than 10ish we'll need to start using background jobs to send them.  Something to keep in mind is that a network request is pending and the 'approve' button is spinning until this completes

Reply:
just thinking outside the box but what if we just didn't await any of the mailgun calls? they could fire off asynchronously without blocking the UI 🤔